docs/reference/project_template.md

```diff
@@ -51,7 +53,37 @@ For example, with a custom template hosted as a GitHub repository, you can use a
 wagtail start myproject --template=https://github.com/githubuser/wagtail-awesome-template/archive/main.zip
 ```
 
-The following is a reference for the default project template.
+You might get an error while trying to generate a custom template. This happens because the `--template` option attempts to parse the templates files in your custom template. To avoid this error, wrap the code in each of your template files with the `{% verbatim %}{% endverbatim %}` tag, like this:
+
+```html
+{% verbatim %}
+{% extends "base.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-blogindexpage{% endblock %}
+
+{% block content %}
+    <h1>{{ page.title }}</h1>
+
+    <div class="intro">{{ page.intro|richtext }}</div>
+
+    {% for post in page.get_children %}
+        <h2><a href="{% pageurl post %}">{{ post.title }}</a></h2>
+        {{ post.specific.intro }}
+        {{ post.specific.body }}
+    {% endfor %}
+
+{% endblock %}
+{% endverbatim %}
+```
+
+The following is a list of custom templates you can use for your projects:
```

Comment:
I think we should maybe link out to Awesome Wagtail and instead add a new section there with Start command templates.

Reply:
@thibaudcolas or @zerolab - thoughts on having an Awesome Wagtail section for Start command templates. Instead of adding them in place in the docs?

Reply:
Probably under Tools. New heading 

`Templates (start command)` 

https://github.com/springload/awesome-wagtail?tab=readme-ov-file#tools

Reply:
Sounds good

Reply:
I've created a [PR](https://github.com/springload/awesome-wagtail/pull/221) to add the `start` command templates to the Awesome Wagtail repo. I will link to it once the PR is accepted and merged.